accounts/forms.py

```diff
@@ -1,9 +1,23 @@
 from django import forms
 from .models import User
+from django.forms import ValidationError
 
 
 class RegistrationForm(forms.ModelForm):
-
     class Meta:
         model = User
-        fields = ['email', ]
+        fields = ['display_name', 'full_name', 'email']
+        widgets = {
+            'email': forms.TextInput(attrs={'class': 'uk-form-width-medium', 'required': 'required'}),
```

Comment:
You want to promote your own stuff, that's fine.  I have enough on my plate already to be having to dig deeper into something else.  I did it the way I did because it was easy and I was able to make it look exactly how I wanted.  I won't theme anymore if that's the case and can revert the changes if you wish.